### PR TITLE
Replace fmt.Errorf with errors.New when there are no values to format…

### PR DESCRIPTION
**Pull request by natemurthy**
The `fmt.Errorf` function is just a convenience wrapper around `errors.New`. If you just want to create an error string with no interpolation why not just use the latter?

### CODE REVIEW

**Comment by neolit123** [comment]
the `fmt.Errorf` -> `errors.New` change is not bad it has performance implications.
but the PR touches other areas especially in kubeadm, please rebase your commits against the latest master.

**Comment by natemurthy** [comment]
@neolit123 Rebased. Any suggestions as to what I should put for release note?

Btw, this is approximately how many other `fmt.Errorf` expressions I could replace with `errors.New`:
```
nmurthy:kubernetes% rg "return fmt.Errorf\(\"(.*?)\"\)" | wc -l
    1692
```

**Comment by neolit123** [comment]
> @neolit123 Rebased. 

thanks but it seems that it still touches other code?

> Any suggestions as to what I should put for release note?

given this isn't user facing there is no need for a release note.

> nmurthy:kubernetes% rg "return fmt.Errorf\(\"(.*?)\"\)" | wc -l
    1692

these are bug numbers, but the problem is that if you submit it all in as a single PR it might take _a while_ before it's merged, unless a top level approver comes by.

File: cmd/kubeadm/app/cmd/upgrade/node.go
Comment (neolit123) [inline cmd/kubeadm/app/cmd/upgrade/node.go]: [1]

File: cmd/kubeadm/app/cmd/upgrade/node.go
Comment (neolit123) [inline cmd/kubeadm/app/cmd/upgrade/node.go]: ^ [1]

File: cmd/kubeadm/app/phases/upgrade/postupgrade.go
Comment (neolit123) [inline cmd/kubeadm/app/phases/upgrade/postupgrade.go]: [1]

File: cmd/kubeadm/app/phases/upgrade/postupgrade.go
Comment (neolit123) [inline cmd/kubeadm/app/phases/upgrade/postupgrade.go]: [1]

File: cmd/kubeadm/app/phases/upgrade/prepull.go
Comment (neolit123) [inline cmd/kubeadm/app/phases/upgrade/prepull.go]: [1]

File: cmd/kubeadm/app/phases/upgrade/staticpods.go
Comment (neolit123) [inline cmd/kubeadm/app/phases/upgrade/staticpods.go]: [1]

File: cmd/kubeadm/app/phases/uploadconfig/uploadconfig.go
Comment (neolit123) [inline cmd/kubeadm/app/phases/uploadconfig/uploadconfig.go]: [1]

File: cmd/kubeadm/app/cmd/upgrade/node.go
Comment (neolit123) [inline cmd/kubeadm/app/cmd/upgrade/node.go]: [1]

**Review by neolit123 — CHANGES_REQUESTED** [review]
/hold
to clarify - please remove code changes unrelated to the PR (marked with [1]). thanks!

**Comment by natemurthy** [comment]
I recently ran a
```
git fetch upstream
git rebase upstream/master
git push -f origin master
```
I don't understand where all these other code changes came from.

**Comment by natemurthy** [comment]
This isn't worth the effort to me any more to constantly keep my fork up-to-date with master upstream.

**Pull request closed without merging**
